marimba-web/Models/Elector.cs

```diff
@@ -19,6 +19,7 @@ public Elector(string name, Marimba.Instrument instrument, MailAddress email, bo
             this.isMembershipPaid = isMembershipPaid;
         }
 
+        // Setters
```

Comment:
Would you be able to make the comments like in Member.cs in the develop branch?  Where you list the arguments and what each does.